lib/cucumber/blendle/steps/model_steps.rb

```diff
@@ -14,7 +14,7 @@
 #     | hello | 100   |
 #     | world | 10    |
 #
-Given(/^the following ((?:(?!should).)+) exist:$/) do |object, table|
+Given(/^the following ([^ ]+) exist:$/) do |object, table|
```

Comment:
The negate of `should` was in there to be non-ambiguous with the paring _Then_ step. The `[^ ]` space negate does the same.

Reply:
And this nexting of regex-groups is no longer supported in cucumber 3.X